packages/presets/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "outDir": "./dist/",
     "noEmit": false
   },
-  "include": ["./src"],
+  "include": ["src/**/*"],
```

Comment:
I think the `include` property are all set to `./src` in all tsconfig files. Why do we need to change for this package?

Reply:
Oh, I will delete this.